spec/models/generic_event/move_approve_spec.rb

```diff
@@ -87,4 +87,46 @@
       expect(generic_event.for_feed).to include_json(expected_json)
     end
   end
+
+  describe '.from_event' do
+    let(:move) { create(:move, date: Date.new(2020, 1, 30)) }
+    let(:event) do
+      create(:event, :approve, :locations, eventable: move,
+                                           details: {
+                                             event_params: {
+                                               attributes: {
+                                                 rejection_reason: 'no_space_at_receiving_prison',
```

Comment:
Don't think these attributes make sense. We want this to include `date` and `create_in_nomis`